omnigibson/action_primitives/starter_semantic_action_primitives.py

```diff
@@ -1426,15 +1445,33 @@ def _get_head_goal_q(self, target_obj_pose):
     def _empty_action(self):
         """
         Get a no-op action that allows us to run simulation without changing robot configuration.
+        This is not a comment on this file, but - let's go to the base copy of the no op action function and update its docstring to say that this is the action that updates the goal to match the current position, and let's add a disclaimer that it might cause drift under external load (e.g. when the controller cannot reach the goal position for some reason).
```

Comment:
Also I meant putting this comment on the compute_no_op_action function on controller_base. For this function too though let's add a docstring that says that this will keep the robot still but aim to move the arms to the saved targets.